docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html

```diff
@@ -63,9 +63,13 @@
     files into an area on the page.
 </p>
 
-<p>
-    Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.
-</p>
+<blockquote>
+    <p><strong>Note</strong>: If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually.</p>
+    <p>
+        Refer to <a href="https://valor-software.com/ng2-file-upload/" target="_blank">ng2-file-upload</a> for complete documentation on this component.
+    </p>
+</blockquote>
```

Comment:
I'm wondering would it be better to have these notes at the top of each of their sections so it's the first thing (except the actual example componenet) that a reader would see? Although for the file upload that would mean having two notes side by side which would look a bit weird so I'm not sure how you'd get around that. Maybe combine the notes into one? What do you think?